chaos-framework/src/main/java/io/openchaos/Arguments.java

```diff
@@ -67,7 +67,7 @@
     @Parameter(names = {
         "-m",
         "--model"}, description = "Test model. Currently queue model and kv model are supported.")
-    public String model = "queue";
+    public String model = "Nacos";
```

Comment:
Please do not modify the default configuration.

Reply:
yes

Reply:
ok，I will revise it.